src/NexmoServiceProvider.php

```diff
@@ -50,6 +51,9 @@ public function register()
         $this->app->singleton(Client::class, function ($app) {
             return $this->createNexmoClient($app['config']);
         });
+        $this->app->singleton(NexmoClient::class, function ($app) {
```

Comment:
Seems to be missing a reference to `Vonage\Client`?

Reply:
`Client::class` is mapped to `Vonage\Client`, and `NexmoClient::class` is mapped to `Nexmo\Client` (lines 5 and 6).